web/yo/app/scripts/services/databaseconnector.js

```diff
@@ -376,16 +376,44 @@ angular.module('oncokbApp')
             }
 
             function updateVUS(hugoSymbol, data, success, fail) {
-                DriveAnnotation
-                    .updateVUS(hugoSymbol, data)
-                    .success(function(data) {
-                        success(data);
-                    })
-                    .error(function(error) {
-                        fail(error);
-                    });
+                if ($rootScope.internal) {
```

Comment:
You actually don't have to wrap it if there is nothing will be executed after `setAPIData('vus', hugoSymbol, data);`, just need to do `return;`

Reply:
I am not sure I understand your point. In my view, it is necessary to do the condition check on internal since we only execute setAPIData() when not in the internal network